src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -1814,75 +1855,549 @@ void GetUserConvectionCoefficients(EnergyPlusData &state)
                                                                  state.dataIPShortCut->cAlphaFieldNames,
                                                                  state.dataIPShortCut->cNumericFieldNames);
         state.dataConvectionCoefficient->InsideFaceAdaptiveConvectionAlgo.Name = state.dataIPShortCut->cAlphaArgs(1); // not used by E+, unique object
-        state.dataConvectionCoefficient->InsideFaceAdaptiveConvectionAlgo.EnteredByUser = true;
-
-        // The following array maps the inputs for the SurfaceConvectionAlgorithm:Inside:AdaptiveModelSelections algorithm input fields
-        // to the corresponding defaults by making a pair with a pointer to the InsideFaceAdaptiveConvectionAlgo algorithm int parameter
-        // to the default int value
-        std::array<int *const, 45> AdaptiveConvectionAlgoInsideDefaults = {
-            &state.dataConvectionCoefficient->InsideFaceAdaptiveConvectionAlgo.SimpleBouyVertWallEqNum,
-            &state.dataConvectionCoefficient->InsideFaceAdaptiveConvectionAlgo.SimpleBouyStableHorizEqNum,
-            &state.dataConvectionCoefficient->InsideFaceAdaptiveConvectionAlgo.SimpleBouyUnstableHorizEqNum,
```

Comment:
This is a funny one. The way this worked before was by looping over this array of raw pointers, which were then each pointing to struct member variables. It was a nice compact way of doing it, but I don't think we want to make using raw pointers a common occurrence. I've eliminated this and listed them out explicitly below; however, this is verbose and adds something like 700-800 lines of code. I'm happy to iterate on this, but it seems like we need an array of references, but I don't think that's allowed. https://stackoverflow.com/a/1164306